modules/landing-page/landing-pricing.tsx

```
@@ -0,0 +1,46 @@
+        <h2 className="mb-8 text-5xl leading-tight font-bold lg:text-6xl">
+          Ready to get
+          <br />
+          <span className="text-gray-400">organized?</span>
+        </h2>
+        <p className="mx-auto mb-12 max-w-2xl text-xl leading-relaxed text-gray-300">
+          Join thousands of individuals and teams who've transformed their productivity with simple,
+          powerful task management.
+        </p>
+
+        <div className="mb-12 flex flex-col items-center justify-center gap-6 sm:flex-row">
+          <button className="group flex items-center rounded-full bg-white px-10 py-5 text-lg font-bold text-black shadow-2xl transition-all duration-300 hover:bg-gray-100">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)